test/test_api_grid.py

```diff
@@ -39,6 +43,91 @@ def test_connected_nodes(self):
         for node_id in IDS:
             self.assertTrue(node_id in response["grid-nodes"])
 
+    def test_model_ids_overview(self):
+        class Net(sy.Plan):
+            def __init__(self):
+                super(Net, self).__init__()
+                self.fc1 = th.nn.Linear(2, 1)
+                self.bias = th.tensor([1000.0])
+                self.state += ["fc1", "bias"]
+
+            def forward(self, x):
+                x = self.fc1(x)
+                return F.log_softmax(x, dim=0) + self.bias
+
+        model = Net()
+        model.build(th.tensor([1.0, 2]))
+        model_ids = ["first_model", "second_model", "third_model"]
+
+        for i in range(1, len(model_ids) + 1):
+            self.my_grid.host_model(model, model_id=model_ids[i - 1])
+            models = json.loads(
```

Comment:
you can make a single request after the for

Reply:
Done! (9b841da)